src/csharp/Grpc.Tools.CSharpGenerator/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>

```

Comment:
See [here](https://msdn.microsoft.com/en-us/library/jj152935(v=vs.110).aspx). Although to be fair, since I'm not bundling the _exe.config_ along with the nuget, this can be omitted. I'm ok going either way.